src/cli.js

```diff
@@ -234,17 +234,24 @@ function cli(inputArgs) {
 
     var args = nopt(knownOpts, shortHands, inputArgs);
 
-    if (args.version) {
-        var cliVersion = require('../package').version;
+    var cliVersion = require('../package').version;
+    // TODO: Use semver.prerelease when it gets released
+    var usingPrerelease = /-nightly|-dev$/.exec(cliVersion);
+    if (args.version || usingPrerelease) {
         var libVersion = require('cordova-lib/package').version;
         var toPrint = cliVersion;
         if (cliVersion != libVersion || /-dev$/.exec(libVersion)) {
             toPrint += ' (cordova-lib@' + libVersion + ')';
         }
-        console.log(toPrint);
-        return Q();
-    }
 
+        if (args.version) {
+            console.log(toPrint);
+            return Q();
+        } else {
+            // Show a warning and continue
+            console.log('Warning: using prerelease version ' + toPrint);
```

Comment:
Can we use logger to print this? This would make it more visible, if we get this printed to console with 'warn' log level.